### ISSUE TITLE
new functionality: MAE split rule like in ExtraTreesRegressor

### ISSUE THREAD

**Issue by Polkas** [ranger/ranger#2190]
New feature development.

Do you think of implementing MAE split rule in contrast to regular MSE (variance) it is much better when we want to e.g. protection against outlier influence. 

Notice: ExtraTreesRegressor sklearn solution is very inefficient. HistGradientBoostingRegressor offers more efficent solution - loss='least_absolute_deviation' although without variable importance option.

source: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesRegressor.html

**Comment by toonn** [comment]
Not sure how this issue ended up on our tracker. We occasionally get bug reports for apache ranger but this is the first scikit-learn issue I've seen.

Sorry we can't help! : )
